src/Model/Item.php

```diff
@@ -1385,6 +1385,19 @@ public static function isValid(array $item)
 			return false;
 		}
 
+		if (!empty($item['uid'])) {
+			$owner = User::getOwnerDataById($item['uid'], false);
+			if (!$owner) {
+				Logger::notice('Missing item user owner data', ['uid' => $item['uid']]);
+				return false;
+			}
+
+			if ($owner['deleted'] || $owner['account_expired'] || $owner['account_removed']) {
```

Comment:
`deleted` is a field in the `contact` table. The other fields look okay.

Reply:
`$owner` comes from the owner view so it would have this field, right? But maybe it isn't relevant?

Reply:
Yes, it is in the view. But it isn't used in the process of deleting users.

Reply:
I'll remove it then.

Reply:
```suggestion
			if ($owner['account_expired'] || $owner['account_removed']) {
```